src/driver/mysql/MysqlDriver.ts

```diff
@@ -217,11 +217,17 @@ export class MysqlDriver implements Driver {
             case ColumnTypes.BOOLEAN:
                 return value === true ? 1 : 0;
             case ColumnTypes.DATE:
-                return moment(value).format("YYYY-MM-DD");
+                if (moment(value).isValid())
+                  return moment(value).format("YYYY-MM-DD");
+                else return '0000-00-00';
```

Comment:
@strictd are you sure `'0000-00-00'` should be persisted here? Maybe `null` should be instead? Because if you want your column to be nullable with this logic it can never be nullable